go/porcelain/deploy.go

```diff
@@ -781,6 +793,7 @@ func bundleFromManifest(ctx context.Context, manifestFile *os.File, observer Dep
 			})
 		}
 
+		// TODO: Move these to the `meta` object above.
```

Comment:
assuming this is a todo in the future

Reply:
Yes, but actually it's not clear we definitely want to do this, so I've removed the comment.